### ISSUE TITLE
How do you apply this?

### ISSUE THREAD

**Issue by src52** [relink2013/Synology_Photos-Tweaks#1]
I have no idea where to place this CSS file. The Readme doesn't specify.

**Comment by dankonig** [comment]
Yes. How do you apply this?

**Comment by Shaxine** [comment]
For now the only way to apply this is through a browser extension to load custom userstyles as per the author instructions: https://www.reddit.com/r/synology/comments/ycvzd5/comment/itqt9hy/?utm_source=share&utm_medium=web2x&context=3

**Comment by relink2013** [comment]
Yup, like @Shaxine already said. As of right now I only recommend using a browser extension. 

It is possible to install it directly to the NAS by referencing it in the right CSS file, but I really don’t recommend it. I have tried to do it before but I never got it to load correctly. Plus package updates will inevitably break it. My last attempt caused me to have to reinstall Photos.
